_includes/contribute-story.html

```diff
@@ -11,5 +11,9 @@
       Contribute a Story
     </a>
   </p>
-  <img class="u-posAbsoluteBottomRight u-size2of5 u-sm-size1of3 u-sm-spaceRight1 u-md-size1of4" src="https://cloudfour.com/wp-content/themes/cloudfour/static/toolkit/images/portland.svg" alt="Portland, Oregon landscape">
+  <img
+    class="u-posAbsoluteBottomRight u-size2of5 u-sm-size1of3 u-sm-spaceRight1 u-md-size1of4"
+    src="data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' viewBox='0 0 140 70' width='140' height='70'%3E%3Cdefs%3E%3Cpolygon id='tree' points='0,70 30,0 60,70'/%3E%3Cuse id='tree-bg' xlink:href='%23tree' fill='%23249579'/%3E%3Cuse id='tree-fg' xlink:href='%23tree' fill='%2335C98D'/%3E%3Cg id='limbs' stroke='%23DBE5EA'%3E%3Cline x1='30' y1='30' x2='30' y2='70'/%3E%3Cline x1='30' y1='45' x2='40' y2='37'/%3E%3Cline x1='21' y1='44' x2='30' y2='50'/%3E%3Cline x1='30' y1='55' x2='40' y2='47'/%3E%3C/g%3E%3Cg id='building'%3E%3Crect x='0' y='0' width='38' height='48' fill='%23D8E5EA'/%3E%3Crect id='building-detail' x='0' y='3' width='38' height='3' fill='%239FB3BE'/%3E%3Cuse xlink:href='%23building-detail' y='6'/%3E%3C/g%3E%3C/defs%3E%3Cuse xlink:href='%23building' x='20' y='22'/%3E%3Cuse xlink:href='%23tree-bg' x='70'/%3E%3Cuse xlink:href='%23building' x='102' y='42'/%3E%3Cg transform='translate(0 14) scale(0.8)'%3E%3Cuse xlink:href='%23tree-fg'/%3E%3Cuse xlink:href='%23limbs' stroke-width='3'/%3E%3C/g%3E%3Cg transform='translate(50 28) scale(0.6)'%3E%3Cuse xlink:href='%23tree-fg'/%3E%3Cuse xlink:href='%23limbs' stroke-width='4' y='5'/%3E%3C/g%3E%3C/svg%3E%0A"
+    alt="Portland, Oregon landscape"
+  >
```

Comment:
@Paul-Hebert Did you try replacing the `<img>` with an `<svg>` instead of making the SVG a data URI?

Reply:
No, but I'd be happy to do that instead if you prefer. I'll do that now.